reconcile/utils/slack_api.py

```diff
@@ -114,24 +120,36 @@ def get_user_id_by_name(self, user_name):
             raise UserNotFoundException(result['error'])
         return result['user']['id']
 
-    def get_channels_by_names(self, channels_names):
-        return {k: v['name'] for k, v in self._get('channels').items()
+    def get_channels_by_names(self, channels_names, limit=CHANNELS_LIMIT):
+        return {k: v['name'] for k, v in self._get('channels',
+                                                   limit=limit).items()
                 if v['name'] in channels_names}
 
-    def get_channels_by_ids(self, channels_ids):
-        return {k: v['name'] for k, v in self._get('channels').items()
+    def get_channels_by_ids(self, channels_ids, limit=CHANNELS_LIMIT):
+        return {k: v['name'] for k, v in self._get('channels',
+                                                   limit=limit).items()
                 if k in channels_ids}
 
-    def get_users_by_names(self, user_names):
-        return {k: v['name'] for k, v in self._get('users').items()
+    def get_users_by_names(self, user_names, limit=USERS_LIMIT):
+        return {k: v['name'] for k, v in self._get('users',
+                                                   limit=limit).items()
                 if v['name'] in user_names}
 
-    def get_users_by_ids(self, users_ids):
-        return {k: v['name'] for k, v in self._get('users').items()
+    def get_users_by_ids(self, users_ids, limit=USERS_LIMIT):
+        return {k: v['name'] for k, v in self._get('users',
+                                                   limit=limit).items()
                 if k in users_ids}
 
     @retry()
-    def _get(self, type):
+    def _get(self, type: str, **kwargs) -> Dict:
```

Comment:
the current consensus is not to use type hints: https://gitlab.cee.redhat.com/service/dev-guidelines/-/merge_requests/76

Reply:
I can remove these if the team has decided to prohibit type annotations.  I'll follow-up with the team to figure out the best path forward for considering an update to this policy.